CMakeLists.txt

```diff
@@ -423,37 +423,30 @@ message(STATUS "BUILD_PLUGINS_DIR set to ${BUILD_PLUGINS_DIR}.")
 # OS specific checks
 TEST_BIG_ENDIAN(BIG_ENDIAN)
 
+# public if we use the headers of the dependency in our public headers
+set(libcsound_PUBLIC_LIBS "")
```

Comment:
There is an guideline for CMake not to use variables like this and just use the targets directly. https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#avoid-custom-variables-in-the-arguments-of-project-commands

Reply:
Ok, replaced with SndFile::sndfile, thanks for the tip!